packages/0xcert-scaffold/src/core/messages.ts

```diff
@@ -0,0 +1,91 @@
+  [ProviderIssue.XCERT_DEPLOY_GATEWAY_INVALID_SIGNATURE]: 'Provided signature is invalid.',
+  [ProviderIssue.XCERT_DEPLOY_GATEWAY_DEPLOY_CANCELED]: 'Order was canceled.',
+  [ProviderIssue.XCERT_DEPLOY_GATEWAY_DEPLOY_ALREADY_PERFORMED]: 'This order was already performed.',
+  [ProviderIssue.XCERT_DEPLOY_GATEWAY_MAKER_NOT_EQUAL_TO_SENDER]: 'Sender is not the creator of this order.',
+  [ProviderIssue.DAPP_TOKEN_NOT_ENOUGH_BALANCE]: 'Sender does not have enough balance.',
+  [ProviderIssue.DAPP_TOKEN_NOT_ENOUGH_ALLOWANCE]: 'You do not have enough allowance.',
+  [ProviderIssue.DAPP_TOKEN_NOT_WHITELISTED_ADDRESS]: 'Tokens cannot be sent to this recipient. Recipients are limited.',
+  [ProviderIssue.DAPP_TOKEN_NOT_MIGRATION_NOT_STARTED]: 'Migration is not in progress.',
+  [ProviderIssue.DAPP_TOKEN_MIGRATION_STARTED]: 'Migration in progress.',
+  [ProviderIssue.DAPP_TOKEN_NOT_ABLE_TO_MIGRATE]: 'Migration cannot be performed. Please notify token owner.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_INVALID_SIGNATURE_KIND]: 'Provided signature kind is invalid.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_TAKER_NOT_EQUAL_TO_SENDER]: 'Sender is not allowed to execute this order.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_CLAIM_EXPIRED]: 'Order has expired.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_INVALID_SIGNATURE]: 'Provided signature is invalid.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_DEPLOY_CANCELED]: 'Order was canceled.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_DEPLOY_ALREADY_PERFORMED]: 'This order was already performed.',
+  [ProviderIssue.ERC20_DEPLOY_GATEWAY_MAKER_NOT_EQUAL_TO_SENDER]: 'Sender is not the creator of this order.',
+  [ProviderIssue.ERC20_TRANSFER_PROXY_TRANSFER_FAILED]: 'Transfer failed.',
+  [ProviderIssue.ACTIONS_GATEWAY_INVALID_SIGNATURE_KIND]: 'Provided signature kind is invalid.',
+  [ProviderIssue.ACTIONS_GATEWAY_INVALID_PROXY]: 'Invalid proxy.',
+  [ProviderIssue.ACTIONS_GATEWAY_SENDER_NOT_A_SIGNER]: 'Sender is not one of the signers.',
+  [ProviderIssue.ACTIONS_GATEWAY_CLAIM_EXPIRED]: 'Order has expired.',
+  [ProviderIssue.ACTIONS_GATEWAY_INVALID_SIGNATURE]: 'Provided signature is invalid.',
+  [ProviderIssue.ACTIONS_GATEWAY_ORDER_CANCELED]: 'Order was canceled.',
+  [ProviderIssue.ACTIONS_GATEWAY_ORDER_ALREADY_PERFORMED]: 'This order was already performed.',
+  [ProviderIssue.ACTIONS_GATEWAY_SIGNERS_DOES_NOT_INCLUDE_SENDER]: 'Sender is not one of the signers.',
+  [ProviderIssue.SIGNER_DOES_NOT_HAVE_ALLOW_CREATE_ASSET_ABILITY]: 'Signer of create asset action does not have allow create asset ability.',
+  [ProviderIssue.SIGNER_DOES_NOT_HAVE_ALLOW_UPDATE_ASSET_ABILITY]: 'Signer of update asset action does not have allow update asset ability.',
+  [ProviderIssue.SIGNER_DOES_NOT_HAVE_ALLOW_MANAGE_ABILITIES_ABILITY]: 'Signer of set asset ledger ability action does not have allow manage abilities ability.',
+  [ProviderIssue.SIGNER_IS_NOT_DESTROY_ASSET_OWNER]: 'Signer of destroy asset action is not the asset owner.',
```

Comment:
[ProviderIssue.SIGNER_IS_NOT_DESTROY_ASSET_OWNER]: 'Signer of DESTROY_ASSET action is not the asset owner.',